src/app/filter/filter.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FilterService {
+  readonly values$: BehaviorSubject<Filters>;
+
+  constructor() {
+    this.values$ = new BehaviorSubject<Filters>({
+      currentPage: 1,
+      entriesPerPage: 20
+    });
```

Comment:
could you move this into the declaration? may you could avoid to use the constructor()

Reply:
Yes, I could do that but I wonder why.
Isn't the constructor's purpose to initialize stuff? xD

Reply:
mhm yep, but if the initialization does not depend on the constructor input parameter we can even avoid it and save some line of code, like your `loading = true` in your app component.

Not sure what is the best practice